### PR TITLE
[RM Ch04] Update Section 4.1 capabilities

### LINKED ISSUE #2398: [RM Ch04] Define missing capabilities in 4.1
Add definitions in Section 4.1.2.2 for  the following:

| Ref | Cloud Infrastructure Capability | Unit | Definition/Notes |
|-- | -- | -- | --|
| e.cap.018 | Simultaneous Multithreading  (SMT) | Yes/No | |
| e.cap.019 | vSwitch Optimisation (DPDK) | Yes/No | |
| e.cap.020	|Storage Encryption | Yes/No | |

Add a **Section 4.1.7 Other Cloud Infrastructure Capabilities**

| Ref | Cloud Infrastructure Capability | Unit | Definition/Notes |
|-- | -- | -- | --|
| o.cap.001 | CPU Architecture | <value> | Values such as x64, ARM, etc. |
| o.cap.002 | Host Operating System | <value> | Values such as Ubuntu20.04, Windows 10 Release #, etc. |
| o.cap.003 | Virtualisation (inc. Containerisation) Software | <value> | Valuse such as KVM, Hyper-V, etc. |

**Edit 4.2: Fill in the Reference Labels**
If PR #2384 merged, then as part of the PR for the above changes
else edit PR #2384 and fill in the references

### PR DESCRIPTION
**Pull request by pgoyal01**
Closes #2398 
Update Section 4.1 with missing capabilities

### CODE REVIEW

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -266,6 +271,15 @@ Table 4-10 shows performance measurement capabilities.
 <p align="center"><b>Table 4-11:</b> Cloud Infrastructure management Resource Management Measurements</p>
 
 
+<a name="4.1.7"></a>
+### 4.1.7 Other Cloud Infrastructure Capabilities
+
+| Ref | Cloud Infrastructure Capability   | Unit/ Value | Definition/Notes |
+|-----|----------------------|---------|-----------------------|
+| o.cap.001 | CPU Architecture | <value> | Values such as x64, ARM, etc. |
```
Comment (karinesevilla) [inline doc/ref_model/chapters/chapter04.md:279]: @pgoyal01 The 'o' reference is not defined. And do we need to introduce it? The capabilities mentioned in 4.1.7 are  required by some workloads and in that sense can by exposed to the workloads.   Can it be renamed  'e.cap'?

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter04.md:279]: @karinesevilla I struggled with this and so Thanks for the feedback. I will move them to e.cap and then populate the tables in Section 4.2.

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter04.md]: @karinesevilla Done. Thanks

**Review by karinesevilla — APPROVED** [review]

File: doc/ref_model/chapters/chapter04.md [hunk 2]
```diff
@@ -167,21 +180,23 @@ This section covers a list of implicit Cloud Infrastructure capabilities and mea
...
+| i.cap.016 | CPU allocation ratio | N:1    | Number of virtual cores per physical core; also known as CPU overbooking ratio. |
+| i.cap.017 | Connection point QoS | Yes/No | QoS enablement of the connection point (vNIC or interface).                     |
 
 <p align="center"><b>Table 4-6:</b> Internal SLA capabilities to Cloud Infrastructure</p>
 
 <a name="4.1.4.3"></a>
-#### 4.1.4.3 Internal Performance Optimisation Capabilities
-**Table 4-7** below shows possible performance optimisation capabilities that can be provided by Cloud Infrastructure. These include capabilities exposed to workloads as well as internal capabilities to Cloud Infrastructure. These capabilities will be determined by the Cloud Infrastructure Profile used by the Cloud Infrastructure.
+#### 4.1.4.3 Internal Performance Optimisation and Security Capabilities
+**Table 4-7** below shows possible performance optimisation and secuirty capabilities that can be provided by Cloud Infrastructure. These include capabilities exposed to workloads as well as capabilities internal to the Cloud Infrastructure. These capabilities will be determined by the Cloud Infrastructure Profile used by the Cloud Infrastructure.
```
Comment (petorre) [inline doc/ref_model/chapters/chapter04.md:191]: "secuirty" -> "security"

**Review by petorre — APPROVED** [review]

**Review by rgstori — APPROVED** [review]

File: doc/ref_model/chapters/chapter04.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter04.md]: @petorre Thanks. Done. Please review.

**Pull request merged**